app/tests/unit/specs/store/connection.spec.js

```diff
@@ -5,6 +5,11 @@ jest.mock(`src/../config.js`, () => ({
   network: `keine-ahnungnet`,
 }))
 
+jest.mock(`@polkadot/util-crypto`)
+import { checkAddress } from "@polkadot/util-crypto"
+
+checkAddress.mockReturnValue([true, 'The address is valid'])
```

Comment:
you're bypassing the actual implementation. why?

Reply:
True! not the best idea ... it was failing for some reason with a correct address.

Reply:
fixed!